server/sonar-server/src/main/java/org/sonar/server/platform/web/RailsAppsDeployer.java

```diff
@@ -87,15 +85,12 @@ static void deployRailsApp(File appsDir, final String pluginKey, ClassLoader app
     if (hasRailsApp(pluginKey, appClassLoader)) {
       LOG.info("Deploying app: " + pluginKey);
       File appDir = new File(appsDir, pluginKey);
-      ClassLoaderUtils.copyResources(appClassLoader, pathToRubyInitFile(pluginKey), appDir, new Function<String, String>() {
-        @Override
-        public String apply(@Nullable String relativePath) {
-          // Relocate the deployed files :
-          // relativePath format is: org/sonar/ror/sqale/app/controllers/foo_controller.rb
-          // app path is: org/sonar/ror/sqale
-          // -> deployed file is app/controllers/foo_controller.rb
-          return StringUtils.substringAfter(relativePath, pluginKey + "/");
-        }
+      ClassLoaderUtils.copyResources(appClassLoader, pathToRubyInitFile(pluginKey), appDir, relativePath -> {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove useless curly braces around statement and then remove useless return keyword [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1602)